### PR TITLE
ci: Spell checking with cspell

### PR DESCRIPTION
**Pull request by TheElectronWill**

### CODE REVIEW

**Comment by titouanj** [comment]
Why not using the official action ? https://github.com/streetsidesoftware/cspell-action

File: .vscode/cspell.json
Comment (titouanj) [inline .vscode/cspell.json:1]: Vscode can use .cspell.json at the root of the project btw

Comment (TheElectronWill) [reply .vscode/cspell.json:1]: Can it use .cspell/cspell.json? I wanted to out everything in one hidden dir

Comment (titouanj) [reply .vscode/cspell.json:1]: Unfortunately that's not possible...

Comment (titouanj) [reply .vscode/cspell.json:1]: What I'm doing is .cspell.json to store the config and .cspell/*.txt to store the custom dicts

Comment (TheElectronWill) [reply .vscode/cspell.json:1]: ah yes, that would work too

**Pull request merged**